code/modules/paperwork/paper.dm

```diff
@@ -724,26 +733,28 @@
 			txt += "Weyland-Yutani Research Grant</H2></center>"
 			txt += "Dear valued researcher. Weyland-Yutani has taken high interest of your recent scientific progress. To further support your work we have sent you this research grant of [grant] credits. Please scan at your local Weyland-Yutani research data terminal to receive the benefits.<BR>\n"
 			txt += "<BR>\n<HR> - <I>Weyland-Yutani</I>"
+		if("ciph_hint")
+			icon_state = "paper_wy_words"
+			name = "Transmission Intercepted"
+			txt += "[MAIN_SHIP_NAME] intelligence Relay</H2></center>"
+			txt += "Progress report<BR>\n\n"
+			txt += "During testing, the theorized component <b>[PROPERTY_CIPHERING]</b> was found to be made of <b>[hint[1]]</b>, Recent discovery made us believe the one of the missing pieces has <b>[isNeutralProperty(hint[3]) ? "neutral" : "negative"]</b> effects.<BR>\n"
+			txt += "This will be the last transmission before the scheduled communications blackout, Testing is successful so far.<BR>\n"
```

Comment:
```suggestion
			txt += "This will be the last transmission before the scheduled communications blackout, testing has been successful so far.<BR>\n"
```